crates/revm/src/db/states/account_status.rs

```diff
@@ -46,7 +27,7 @@ impl AccountStatus {
 
     /// Account was destroyed by calling SELFDESTRUCT.
     /// This means that full account and storage are inside memory.
-    pub fn was_destroyed(&self) -> bool {
+    pub fn is_destroyed(&self) -> bool {
```

Comment:
`was_destroyed` is more accurate as it contains `DestroyedChanged`

Reply:
ah, right, good point